db/migrate/20201210032852_discourse_github_rebuild_git_history.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DiscourseGithubRebuildGitHistory < ActiveRecord::Migration[6.0]
+  def up
+    execute "DELETE FROM github_commits"
```

Comment:
This is a little confusing to me - the migration deletes all commits in the database? Is the idea that they will be rebuilt shortly afterwards?

Reply:
Yes, exactly. They will be rebuilt the next time the `DiscourseGithubPlugin::UpdateJob` runs.

Reply:
I assume there's going to be a small amount of time between that where things will be a bit broken? That is probably OK if not too long.

I would recommend adding a comment here saying they will be rebuilt in case anyone else is confused.

Reply:
The commits inside that table are not used for anything else besides granting badges - every 4 hours the `UpdateJob` fetches new commits, saves them in the database and grants badges to users who qualify. If the commits disappear, users will still keep their badges.

I will add a comment to clear this up 👍

Reply:
I've added a comment on the migration file.